File: app/src/main/java/com/sd/demo/tag_view/item/TestItem.java
package com.sd.demo.tag_view.item;

import android.util.Log;

import com.sd.lib.tag_view.ITagView;

public class TestItem implements ITagView.Item
{
    private static final String TAG = TestItem.class.getSimpleName();

    @Override
    public void init(ITagView tagView)
    {
        Log.i(TAG, "init:" + tagView + " " + this);
    }

    @Override
    public void destroy()
    {
        Log.i(TAG, "destroy " + " " + this);
    }
}

File: app/src/main/java/com/sd/demo/tag_view/item/TestItem.kt
package com.sd.demo.tag_view.item

import android.util.Log
import com.sd.lib.tag_view.ITagView

class TestItem : ITagView.Item {

    override fun initItem(tagView: ITagView) {
        Log.i(TestItem::class.java.simpleName, "initItem:$tagView $this")
    }

    override fun destroyItem() {
        Log.i(TestItem::class.java.simpleName, "destroyItem $this")
    }
}